packages/sources/coinmarketcap/package.json

```diff
@@ -33,5 +34,8 @@
     "start": "yarn server:dist"
   },
   "devDependencies": {},
-  "dependencies": {}
+  "dependencies": {
```

Comment:
I don't think we _need_ these, but might be nice for documenting dependencies?

@HenryNguyen5 What do you think here? How should we best handle intra-mono-repo dependencies?

Reply:
This is the right way to do it, packages should be treated as individual units, the notion of monorepo is purely just for us. If we ever publish these packages (which we should), then people trying to download these packages and use them would be unable to since theyre missing deps. These packages should ideally be versioned properly too, but thats something to address later on